protocol/app/ante/sigverify.go

```diff
@@ -100,10 +99,14 @@ func (svd SigVerificationDecorator) AnteHandle(
 		// `GoodTilBlock` for replay protection.
 		if !skipSequenceValidation {
 			if accountpluskeeper.IsTimestampNonce(sig.Sequence) {
-				start := time.Now()
+				defer telemetry.ModuleMeasureSince(
```

Comment:
This needs to be put within ProcessTimestampNonce so that the `defer` clause is run when `ProcessTimestampNonce` returns. This is a common pattern to measure the latency of a function from start to finish

Reply:
```
				defer telemetry.ModuleMeasureSince(
					accountplustypes.ModuleName,
					time.Now(),
					metrics.TimestampNonce,
					metrics.Latency,
				)
				err = svd.akp.ProcessTimestampNonce(ctx, acc, sig.Sequence)
```
A bit confused. As u said defer run after the function below is run. In this case it is right above 
`				err = svd.akp.ProcessTimestampNonce(ctx, acc, sig.Sequence)
`
So it will run after ProcessTimestampNonce?

Reply:
A defer statement defers the execution of a function until the surrounding function returns. https://go.dev/tour/flowcontrol/12